BUILD.md

```diff
@@ -58,7 +58,19 @@ make release-controller \
   IMAGE=<your_registry>/<your_controller_image> \
   VERSION=<your_image_tag>
 ```
-If `VERSION` is not specified, the Makefile will auto-generate one for you.
+
+Update the `IMAGE` variable to reference the image registry you'd like to push
+your image to. You must have write permissions on the registry. The `VERSION`
+variable is useful for versioning your image with custom tag. If `VERSION` is
+not specified, the Makefile will auto-generate one for your image.
```

Comment:
Like `VERSION`, we should also add a statement about the default `IMAGE` value if not specified

Reply:
I think highlighting it for `VERSION` was important because it's optional and things wouldn't fail if VERSION is not specified. So if things didn't fail and a new tag was auto generated its important that we mention that. 
If image is not specified things would just fail. I will Ivan comment.

Reply:
@PrasadG193 @viveksinghggits I can't push to this branch anymore. Either we merge this PR as-is, or can you apply this diff to the branch?

````diff
diff --git a/BUILD.md b/BUILD.md
index 66586c3c..44381834 100644
--- a/BUILD.md
+++ b/BUILD.md
@@ -60,9 +60,11 @@ make release-controller \
```
 
 Update the `IMAGE` variable to reference the image registry you'd like to push
-your image to. You must have write permissions on the registry. The `VERSION`
-variable is useful for versioning your image with custom tag. If `VERSION` is
-not specified, the Makefile will auto-generate one for your image.
+your image to. You must have write permissions on the registry. If `IMAGE` is
+not specified, the Makefile will use the default of `kanisterio/controller`.
+
+The `VERSION` variable is useful for versioning your image with custom tag. If
+`VERSION` is not specified, the Makefile will auto-generate one for your image.
 
 For example, the following command will build and push your image to the
 registry at `ghcr.io/myregistry/kanister`, with the tag `20221003`:
````

Reply:
```suggestion
Update the `IMAGE` variable to reference the image registry you'd like to push
your image to. You must have write permissions on the registry. If `IMAGE` is
not specified, the Makefile will use the default of `kanisterio/controller`.

The `VERSION` variable is useful for versioning your image with a custom tag.
If `VERSION` is not specified, the Makefile will auto-generate one for your
image.
```

Reply:
Done ✅ Thank you 👍🏼